contracts/KyberNetwork.sol

```diff
@@ -86,7 +89,11 @@ contract KyberNetwork is Withdrawable, Utils2, KyberNetworkInterface {
         tradeInput.walletId = walletId;
         tradeInput.hint = hint;
 
-        return trade(tradeInput);
+        uint actualDestAmount = trade(tradeInput);
```

Comment:
I think modifier will be nicer solution. need to read about it though. I don't think return value is an issue.